playback/src/main/kotlin/voice/playback/di/PlaybackModule.kt

```diff
@@ -86,18 +88,19 @@ object PlaybackModule {
     scope: CoroutineScope,
     sleepTimer: SleepTimer,
     sleepTimerCommandUpdater: SleepTimerCommandUpdater,
+    playStateManager: PlayStateManager,
   ): MediaLibraryService.MediaLibrarySession {
     return MediaLibraryService.MediaLibrarySession.Builder(service, player, callback)
       .setSessionActivity(mainActivityIntentProvider.toCurrentBook())
       .build()
       .also { session ->
         scope.launch {
-          sleepTimer.leftSleepTimeFlow
-            .map { it != Duration.ZERO }
-            .distinctUntilChanged()
-            .collect { sleepTimerActive ->
-              sleepTimerCommandUpdater.update(session, sleepTimerActive)
-            }
+          merge(
```

Comment:
I don't think merge is correct here. The latest value is not necessarily the right one. Probably a combine and then conditional logic makes more sense